examples/pulse-mpi.py

```diff
@@ -61,14 +61,20 @@
 
 
 @mpi_entry_point
-def main(ctx_factory=cl.create_some_context):
+def main(ctx_factory=cl.create_some_context, use_leap=False):
     """Drive the example."""
     cl_ctx = ctx_factory()
     queue = cl.CommandQueue(cl_ctx)
     actx = PyOpenCLArrayContext(queue,
                 allocator=cl_tools.MemoryPool(cl_tools.ImmediateAllocator(queue)))
 
     logger = logging.getLogger(__name__)
+    
+    import importlib
+    leap_spec = importlib.util.find_spec("leap")
+    not_found = leap_spec is None
+    if not_found:
+        raise ValueError("Leap uninstalled")
```

Comment:
Why have these snippets?
- The examples *must* work without leap.
- `ModuleNotFoundError: No module named 'leap'` is already pretty explicit.

Reply:
Yeah, this was dumb - I've removed it with 2caff69.